_security-plugin/configuration/yaml.md

```diff
@@ -124,6 +124,12 @@ plugins.security.restapi.password_validation_regex: '(?=.*[A-Z])(?=.*[^a-zA-Z\d]
 plugins.security.restapi.password_validation_error_message: "Password must be minimum 8 characters long and must contain at least one uppercase letter, one lowercase letter, one digit, and one special character."
 ```
 
+You can set the `plugins.security.allow_default_init_securityindex` property to `true` to have the security plugin automatically initialize with default settings in the config folder if the security index fails to initialize when OpenSearch launches. By default, the setting is `false`.
```

Comment:
Just want to make sure the comment here conveys the right message. When this setting is true, security plugin will attempt to create security index and default configs if they are absent.